docs/develop/java/testing-suite.mdx

```diff
@@ -280,6 +281,12 @@ Mock the Activity invocation when unit testing your Workflows.
 
 When integration testing Workflows with a Worker, you can mock Activities by providing mock Activity implementations to the Worker.
 
+### How to mock Nexus Operations {#mock-nexus-operations}
+
+Mock the Nexus Operation invocation when unit testing your Workflows.
+
+When integration testing Workflows with a Worker, you can mock Nexus Operations by providing mock Nexus Service implementations to the Worker.
```

Comment:
```suggestion
When integration testing Workflows with a Worker, you can mock Nexus Operations by providing a mock Nexus Service implementations to the Worker.
```

Reply:
To make the sentence singular it would read `you can mock a Nexus Operation by providing a mock Nexus Service implementation to the worker` but I don't like that phrasing as much because it doesn't match what we have for activities.